packages/cli/src/cli.js

```diff
@@ -38,14 +35,6 @@ export function init(cb) {
     /**
      * OPTIONS
      */
-    .option('gas-price', {
```

Comment:
Removes cli global options. Looking forward on the power-cli the powerusers are not going to customize that much their environment and are going to mostly interact with the default environments. 

A future step is adding a new command to setup new environments once. Afterwards those will be available and the toolkit will load them automatically for future requests.